tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,35}-django111-drf{37,38,39,latest},py35-django{20,21,22}-drf{37,38,39,latest}
+envlist = py35-django{111,20,21,22}-drf{37,38,39,latest}
```

Comment:
This might be a good time to drop the DRF 3.7 and 3.8 tests, and add 3.10 (since edx-platform is already on 3.9.4).